docs/proposals/20220825-azuremanaged-cluster-exp-graduation.md

```diff
@@ -0,0 +1,126 @@
+---
+title: AzureManagedCluster graduation from experimental
+authors:
+  - "@jackfrancis"
+reviewers:
+  - @CecileRobertMichon
```

Comment:
did everyone listed in reviewers actually review? if not let's remove those who didn't prior to merge (or get their review)

Reply:
@zmalik @NovemberZulu @mtougeron could you kindly review this and weigh in?

If we don't hear from folks in ~a week or so let's remove those names from this list and finalize for merge.

Reply:
I'll take a look by EOD Monday.